test/remote/README.md

```diff
@@ -18,6 +18,20 @@ kubectl create ns argocd-e2e
 kubectl -n argocd-e2e apply -f <your Argo CD installation manifests>
 ```
 
+Argo CD installation manifest, for example `argocd-test.yaml`
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: argocd-test
+  namespace: argocd-e2e
+spec:
+  server:
+    route:
+      enabled: true
+```
+
```

Comment:
This is not an Argo CD installation manifest, but an Argo CD Operator's operand. I think in the long run, this may be confusing for users wanting to test this (because it will be useless and a no-op without the Operator).

Reply:
Please add some prosa around and label it clearly, e.g. "if you want to test with the Argo CD Operator, ..."

Reply:
> This is not an Argo CD installation manifest, but an Argo CD Operator's operand.
 got it.

> I think in the long run, this may be confusing for users wanting to test this (because it will be useless and a no-op without the Operator).

As per our requirement we are using this operand yaml for downstream testing. However for the step ```kubectl -n argocd-e2e apply -f <your Argo CD installation manifests>``` I feel we should let the user know what can be variety of manifest files with examples.

Like Argo CD testing (installed through GotOps operator) we are using this Argo CD Operator's operand. Would you please point out what other possibilities are ?

Reply:
The other possibilities are the same as for basic Argo CD installation, e.g. use `install.yaml` from Argo CD repository or installation via Helm chart. The important thing is, that the Argo CD you want to test gets installed on the cluster.

Reply:
> The important thing is, that the Argo CD you want to test gets installed on the cluster.
Make sense.
My intention is to provide the detailed information in terms of an example. User can install install Argo CD from repository itself or may be through an operator like argocd operator, gitops operator. I have update the example description, please have look